### PR TITLE
#1569 Pickers: Adds the ability to disable picker controls

### LINKED ISSUE #1569: TagPicker - Add the ability to disable the control
It would be great if a TagPicker component could be disabled. Thanks!

Ladislau

### PR DESCRIPTION
**Pull request by aleksandrjPersonal**
#### Pull request checklist

- [x ] Addresses an existing issue: Fixes #1569 
- [ ] Include a change request file if publishing 
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes

#1569 Pickers: Adds the ability to disable picker controls

#### Focus areas to test

Pickers

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/pickers/AutoFill/BaseAutoFill.tsx [hunk 1]
```diff
@@ -100,6 +100,7 @@ export class BaseAutoFill extends BaseComponent<IBaseAutoFillProps, IBaseAutoFil
       onChange={ this._onChange }
       onKeyDown={ this._onKeyDown }
       onClick={ this._onClick }
+      disabled={ this.props.disabled }
```
Comment (leddie24) [inline packages/office-ui-fabric-react/src/components/pickers/AutoFill/BaseAutoFill.tsx]: You can remove this, it's inherited inside of getNativeProps

**Comment by aleksandrjPersonal** [comment]
@micahgodbolt sure :)

**Comment by aleksandrjPersonal** [comment]
@micahgodbolt checkboxes added to demonstrate the disabling of the tag and Document pickers

File: packages/office-ui-fabric-react/src/components/pickers/BasePicker.Props.ts [hunk 2]
```diff
@@ -49,6 +49,10 @@ export interface IBasePickerProps<T> extends React.Props<any> {
    * @default undefined
    */
   inputProps?: React.HTMLProps<HTMLInputElement>;
+  /**
+   * Flag for disabling the picker.
+   */
```
Comment (mikewheaton) [inline packages/office-ui-fabric-react/src/components/pickers/BasePicker.Props.ts]: Add `@default false` here so that we're clear in the documentation.

**Pull request merged**